### PR TITLE
Emphasize changing server host url when deploying

### PR DESCRIPTION
**Pull request by joelostblom**
This is branched from #1083, so please merge that first (I didn't want to add to it since it was already approved).

@rpkyle Only the two last commits are new here emphasizing the need to specify `host = '0.0.0.0'` since this is different from running locally. I also removed the port argument [since it is just set to the default for app$run_server](https://github.com/plotly/dashR/blob/add8dd3d93b23c0feb966e39f711ed58be256373/R/dash.R#L1183)

Post-merge checklist:

The master branch is auto-deployed to `dash.plotly.com`.
Once you have merged your PR, wait 5-10 minutes and check dash.plotly.com
to verify that your changes have been made.

- [x] I understand

### CODE REVIEW

**Comment by joelostblom** [comment]
Rebased on master and tweaked wording slightly for clarity.

File: dash_docs/chapters/deployment/index.R [hunk 1]
```diff
@@ -93,7 +93,7 @@ app$callback(output=list(id='display-value', property='children'),
   }
 )
 
-app$run_server(host = '0.0.0.0', port = Sys.getenv('PORT', 8050))
+app$run_server(host = '0.0.0.0')
```
Comment (rpkyle) [inline dash_docs/chapters/deployment/index.R:96]: Nice catch! We already populate `port` if unspecified within `run_server`, so this can be removed.

**Review by rpkyle — APPROVED** [review]
@joelostblom Thanks again for this one, just one minor comment to address but will approve since it's an easy fix and shouldn't take you more than 15 seconds. Cheers! :woman_dancing:

**Comment by joelostblom** [comment]
Thanks for catching the typo @rpkyle, updated!

**Comment by joelostblom** [comment]
Rebasing on master, one sec

**Comment by rpkyle** [comment]
@joelostblom There's a spurious visual diff that can't possibly be caused by your minor edits, so I'm going to use my magic wand to merge anyhow. I suspect this is caused by a nearly rendered index page, but will sort that out later. Thanks! :woman_dancing:

**Pull request merged**